common/src/main/java/io/netty/util/concurrent/DefaultThreadFactory.java

```diff
@@ -84,7 +82,7 @@ private static String toPoolName(Class<?> poolType) {
         }
     }
 
-    public DefaultThreadFactory(String poolName, boolean daemon, int priority, ThreadGroup threadGroup) {
+    public DefaultThreadFactory(String poolName, boolean daemon, int priority) {
```

Comment:
@jasontedor maybe we could just keep this constructor and handle the case special where it is `null` (which will be the default) ?